### DISCUSSION TITLE
Webxr not working on android chrome

### DISCUSSION THREAD

**Discussion by challtshan** [google/model-viewer#2788] [Q&A]
### Description
I tried using webxr on android chorme v.93 (with project that use ReactJS)
`ar-modes = "webxr" `

But, when i open on android chrome, AR mode button is missing.

* I use webxr because I need to display hotspot 

#### Live Demo
```
<model-viewer
                  src=url
                  ios-src=url
                  ar
                  ar-modes="webxr"
                  camera-controls >
....
</model-viewer>
```

### Version
- model-viewer: v. 1.2.1

### Browser Affected
- [ ] Chrome, version: 93.0.4577.62

### OS
- [ ] Android

### AR
- [ ] WebXR

**Comment by elalish** [comment]
I can't repro. What phone are you using? Not all  are ARCore-capable, though  most are. Also can  you paste any relevant  messages  from the  console here?

**Comment by challtshan** [comment]
I use Poco F3 and mi mix 2s

I think there is a problem from my side.

I use this demo site to create example code.
https://modelviewer.dev/editor/

Here a model viewer section.

```
<model-viewer src="Astronaut.glb" poster="poster.webp" ar ar-modes="webxr" camera-controls shadow-intensity="1">
    <button class="Hotspot" slot="hotspot-1" data-position="0.04623932060614927m 1.6916264790857483m 0.3276764016232603m" data-normal="0.2586151003950921m -0.19952579310771987m 0.9451495583939994m" data-visibility-attribute="visible">
        <div class="HotspotAnnotation">test</div>
    </button>
    <div class="progress-bar hide" slot="progress-bar">
        <div class="update-bar"></div>
    </div>
    <button slot="ar-button" id="ar-button">
        View in your space
    </button>
    <div id="ar-prompt">
        <img src="https://modelviewer.dev/shared-assets/icons/hand.png">
    </div>
</model-viewer>
```

I run this code in https://modelviewer.dev/editor/
It has AR mode button. But, after i run this code with my project. AR mode button is missing.

Is any configuration I missed?

**Comment by elalish** [comment]
That all looks fine; I bet if you look at the JS console (using chrome://inspect) you'll find a useful message. Most likely you're serving  on  HTTP instead of HTTPS, which is  required for  WebXR. Or possibly you rejected the WebXR permissions once and it's not asking you anymore (you can manually revert them).
